web_responsive/static/src/scss/web_calendar.scss

```
@@ -0,0 +1,19 @@
+.o_calendar_mobile_sidebar_button {
+    display: none;
+}
+@media (max-width: 767.98px) {
```

dùng hàm thay cho việc gán cứng px
```suggestion
@include media-breakpoint-down(sm) {
```